WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -231,7 +258,7 @@ extension SiteInformationWizardContent: UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return Rows.count()
+        return Row.count()
```

Comment:
Rows worked better here but Row works better for everything else. Just a note.

Reply:
Yeah, I thought the same thing.

The reason I changed it anyway is because enums are normally singular, and additionally this would normally be:

```swift
return Row.allCases.count
```

which makes it ok that it's singular.  I can change this line to be as described if you prefer.

Reply:
I think `Row.allCases.count` is better